stores-api.yaml

```diff
@@ -1,95 +1,602 @@
+        h:
+          type: integer
+          format: int32
+        w:
+          type: integer
+          format: int32
+    Price:
+      title: Price
+      type: object
+      properties:
+        currencyCode:
+          type: string
+        value:
+          type: number
+    Product:
+      title: Product
+      type: object
+      properties:
+        description:
+          $ref: '#/components/schemas/LocalizedString'
+        id:
+          type: string
+        masterVariant:
+          $ref: '#/components/schemas/ProductVariant'
+        name:
+          $ref: '#/components/schemas/LocalizedString'
+        price:
+          $ref: '#/components/schemas/Price'
+        slug:
+          $ref: '#/components/schemas/LocalizedString'
+      example:
+        id: 6a19bf78-dce2-4830-845f-b029ada5472e
+        name:
+          de_DE: 2x kleine Hamamtücher 50x90cm
+        description:
+          de_DE: |-
+            Du bekommst 2x kleine Hamamtücher 50x90cm aus 100% Baumwolle perfekt im Bad oder Küche
+            Voraussichtliche Lieferung: ab sofort
+        slug:
+          en_US: 2-hamam-50-90cm-rot-weiss
+          de_DE: 2x-kleine-hamamtuecher-50x90cm
+        price:
+          value: 25
+          currencyCode: EUR
+        masterVariant:
+          assets: []
+          attributes: []
+          id: 1
+          images:
+          - url: https://7cbf5172f31e91cb7a5d-d8e96be60ea1fd7ad98fb07a482142ee.ssl.cf3.rackcdn.com/2-hamam-50-90cm-rot--q9KcG8UT.jpg
+            dimensions:
+              w: 566
+              h: 566
+            label: 2 Hamamtücher 50 x 90cm
+          key: 2-hamam-50-90cm-rot-weiss
+          sku: 2-hamam-50-90cm-rot-weiss
+    ProductDraft:
+      title: ProductDraft
+      type: object
+      properties:
+        description:
+          $ref: '#/components/schemas/LocalizedString'
+        name:
+          $ref: '#/components/schemas/LocalizedString'
+        price:
+          $ref: '#/components/schemas/Price'
+        slug:
+          $ref: '#/components/schemas/LocalizedString'
+      example:
+        name:
+          de_DE: 2x kleine Hamamtücher 50x90cm
+        description:
+          de_DE: |-
+            Du bekommst 2x kleine Hamamtücher 50x90cm aus 100% Baumwolle perfekt im Bad oder Küche
+            Voraussichtliche Lieferung: ab sofort
+        slug:
+          en_US: 2-hamam-50-90cm-rot-weiss
+          de_DE: 2x-kleine-hamamtuecher-50x90cm
+        price:
+          value: 25
+          currencyCode: EUR    
+    ProductVariant:
+      title: ProductVariant
+      type: object
+      properties:
+        attributes:
+          type: array
+          items:
+            $ref: '#/components/schemas/Attribute'
+        id:
+          type: integer
+          format: int32
+        images:
+          type: array
+          items:
+            $ref: '#/components/schemas/Image'
+        sku:
+          type: string
+    LocalizedString:
+      title: LocalizedString
+      type: object
```

Comment:
Could you add a definition for a LocalizedString and/or an example?

Reply:
Fixed in new commit (but it doesn't render for me :/ )

Reply:
There is an example contained in the overall example for the higher level object, so it's all good.